apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,68 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  // Demo saveConclusion function
+  const saveConclusion = () => {
+    // Demo getRegistryAnswer
+    const getRegistryAnswer = (id) => {
+      // How do we get the registry answer?
+      return "true";
+    };
+
+    // Demo getQuestionAndAnswers
+    const getQuestionAndAnswers = () => {
+      return this.stack.reduce((acc, question) => {
+        acc.push({
+          ...question,
+          // id: does `question` contain `id`? (Yes)
```

Comment:
Yeah, for now this is the (frequently changing) id from FL. In the future we'll use our own id's.

Reply:
How do we handle skipped questions? When we save the session i think we have to save them too

Reply:
@afjlambert, ah okay, so if we use our own id's, that will be great. That fixes the problem in the above comment as well. We're going to need that for this API (-:

@svenjens, we don't need to store skipped questions. With the functions we made for Refresh, we can let the checker go through the checker with provided answers and make a new route